allennlp/data/instance.py

```diff
@@ -45,12 +45,16 @@ def get_padding_lengths(self) -> Dict[str, Dict[str, int]]:
             lengths[field_name] = field.get_padding_lengths()
         return lengths
 
-    def as_array(self, padding_lengths: Dict[str, Dict[str, int]]) -> Dict[str, DataArray]:
+    def as_arrays(self, padding_lengths: Dict[str, Dict[str, int]] = None) -> Dict[str, DataArray]:
```

Comment:
Hmm, this doesn't have the same api as for `Fields` though - maybe the methods on `Instance` and `Dataset` should be called `as_array_dict`?